src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/ResponseRejectionContentProvider.java

```diff
@@ -1,7 +1,7 @@
 package uk.gov.hmcts.cmc.claimstore.documents.content;
 
 import org.springframework.stereotype.Component;
-import uk.gov.hmcts.cmc.ccd.deprecated.domain.CCDYesNoOption;
+import uk.gov.hmcts.cmc.ccd.domain.CCDYesNoOption;
```

Comment:
This looks wrong usage of `CCDYesNoOption` in core module. It should be `YesNoOption`